cypress/integration/spec.js

```diff
@@ -0,0 +1,34 @@
+/// <reference types="Cypress" />
+
+import { SearchPage } from "../page-objects/search-page"
+
+// @ts-check
+const searchPage = new SearchPage()
+describe('Article search', () =>{
+  beforeEach(() =>{
+    searchPage.navigateToSearchPage()
+    searchPage.search("catt")
+  })
+
+  it('search should show results', () => {
+    cy.get('.result').first()
+      .should('have.text', "Cattaraugus County, New YorkCounty in New York")
+  })
+
+  it('results with image should show image', () => {
+    cy.get('.result').first()
+      .children().should('have.length', 2)
+  })
+
+  it('results without image should not show image', () => {
+    cy.get('.result').first().next()
+      .children().should('have.length', 1)
+  })
+  
+  it('article should open from search results page', () => {
+    cy.get('.result').first().next()
+    .children().should('have.length', 1)
+    cy.get('body').type('{enter}').type('{downarrow}').type('{enter}')
```

Comment:
Those keys are going to be used for all interactions. It might be worth encapsulating them in a specific class similar to a page object. Doesn't have to be in the PR, just something to consider instead of retyping the same kind of line everywhere.

Reply:
fixed